plugins/module_utils/sops.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c), Edoardo Tenani <e.tenani@arduino.cc>, 2018-2020
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+ __metaclass__ = type
```

Comment:
I have no idea where that space came from :)
```suggestion
__metaclass__ = type
```